reana_job_controller/config.py

```diff
@@ -80,3 +84,9 @@
 """Kerberos configMap name. Must be the same as in
 reana_cluster/backends/kubernetes/templates/configmaps/kerberos.yaml.
 """
+
+SLURM_CLUSTER_HOSTNAME = os.getenv('SLURM_HOSTNAME', '188.185.70.122')
+"""Hostname of SLURM submit node."""
+
+SLURM_CLUSTER_PORT = os.getenv('SLURM_CLUSTER_PORT', '32769')
```

Comment:
What do you think of adding `ssh` in the name of this variable and `headnode` to the previous one (`SLURM_CLUSTER_HOSTNAME `)?

So we state more explicitly how we connect to Slurm, "`ssh`ing" to `headnode`, using `SLURM_CLUSTER_HEADNODE_HOSTNAME` through `SLURM_CLUSTER_HEADNODE_SSH_PORT`.

Reply:
I would go for `SLURM_HEADNODE_HOSTNAME`, `SLURM_HEADNODE_PORT` and add a more verbose description.